sample-service/pom.xml

```diff
@@ -22,6 +22,48 @@
             <artifactId>azure-keyvault-secrets-spring-boot-starter</artifactId>
         </dependency>
 
+        <!--Security below-->
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-web</artifactId>
+        </dependency>
```

Comment:
I believe this comes from the parent and can be removed here (also, it's not a security dependency - per the note above).